### PR TITLE
docs: normalize blocks

### PR DESCRIPTION
**Pull request by Shinigami92**
1. this normalized the docs by following the schema suggested: https://vitepress.dev/guide/markdown#default-title
2. it alters some headlines slightly

### CODE REVIEW

File: docs/guide/upgrading.md [hunk 1]
```diff
@@ -4,19 +4,17 @@ This is the migration guide for upgrading from v7 to v8.
...
 
-::: info
-Not the version you are looking for?
+::: info Not the version you are looking for?
 
 - [Upgrading to v7](https://v7.fakerjs.dev/guide/upgrading.html)
 - [Upgrading to v6](https://v6.fakerjs.dev/migration-guide-v5/)
-
-:::
+  :::
```
Comment (ST-DDT) [inline docs/guide/upgrading.md]: This looks wrong to me.

File: docs/guide/localization.md [hunk 2]
```diff
@@ -28,12 +28,12 @@ import { faker } from '@faker-js/faker/locale/de';
 
 This will then just load the German locales with additional English locales as fallback. The fallback is required due to not all locales containing data for all features. If you encounter a missing locale entry in your selected language, feel free to open a Pull Request fixing that issue.
 
-::: info
+::: warning Attention
```
Comment (ST-DDT) [inline docs/guide/localization.md]: Not sure this warrants a warning. What do the others think?

grafik

Comment (matthewmayer) [reply docs/guide/localization.md]: Agree. Think it can just be incorporated as regular text.

Comment (Shinigami92) [inline docs/guide/localization.md]:
```suggestion
::: info
```

**Review by ST-DDT — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

**Pull request merged**